crowd_anki/anki_exporter.py

```diff
@@ -4,10 +4,10 @@
 
 import anki.utils
 
-from thirdparty.pathlib import Path
+from .thirdparty.pathlib import Path
```

Comment:
You forgot to do aliasing here. Also instead of doing it everywhere - you can do it in one place (i.e. thirdparty/pathlib.py) and then import it from there

Reply:
Thanks, I tried that, the problem was that it covered the default library with pathlib.py. 
This is why I created a pathlib_wrapper in the utils folder.